doc/man3/SSL_load_client_CA_file.pod

```diff
@@ -66,6 +67,20 @@ The operation failed, check out the error stack for the reason.
 
 Pointer to the subject names of the successfully read certificates.
 
+
```

Comment:
You'll need to add a line with `=back` above this

Reply:
Done. Thanks!